internal/commands/build/compose.go

```diff
@@ -0,0 +1,65 @@
+package build
+
+import (
+	"fmt"
+	"path"
+	"reflect"
+
+	"github.com/docker/app/types"
+	"github.com/docker/buildx/build"
+	"github.com/docker/cli/cli/compose/loader"
+	compose "github.com/docker/cli/cli/compose/types"
+)
+
+// parseCompose do parse app compose file and extract buildx Options
+// We don't rely on bake's ReadTargets + TargetsToBuildOpt here as we have to skip environment variable interpolation
+func parseCompose(app *types.App, options buildOptions) (map[string]build.Options, error) {
+
+	// Fixme can have > 1 composes ?
```

Comment:
do we have the answer?

Reply:
no. So the Fixme

Reply:
We should probably just remove this option, and wait for `overrides` to be implemented in a future spint.